bundles/automation/org.eclipse.smarthome.automation.module.core/ESH-INF/automation/moduletypes/ItemTriggers.json

```diff
@@ -128,6 +138,40 @@
                "required":true
             },
             {  
+               "name":"oldState",
```

Comment:
oldState does not make sense on an ItemStateEvent, as there is no old value - if someone is interested in a change, he must use ItemStateChangedEvent.